.devcontainer/stack-config.yaml

```diff
@@ -0,0 +1,25 @@
+nix:
+  enable: true
+  packages: [glpk, pcre, zlib, glibc]
+
+# This file contains default non-project-specific settings for 'stack', used
+# in all projects.  For more information about stack's configuration, see
+# http://docs.haskellstack.org/en/stable/yaml_configuration/
+
+# The following parameters are used by "stack new" to automatically fill fields
+# in the cabal config. We recommend uncommenting them and filling them out if
+# you intend to use 'stack new'.
+# See https://docs.haskellstack.org/en/stable/yaml_configuration/#templates
+templates:
+  params:
+    author-name: Pavel Shirshov
```

Comment:
Can you change these to placeholders? I want to keep this repo as a reference and submit the updated version to the VS code gallery.

Reply:
done